sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -341,6 +329,7 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
             try {
                 processStatusUpdate(status);
                 reconciler.update(status);
+                TaskKiller.update(status);
```

Comment:
Is there some way to register components that need to have `update` called here, instead of having them added manually?

Reply:
Good idea, out of scope.  PRs welcome.